cmd/osctl/cmd/cluster.go

```diff
@@ -110,6 +116,78 @@ func create(ctx context.Context) (err error) {
 
 	defer provisioner.Close() //nolint: errcheck
 
+	b64Configs := map[string]string{"Init": "", "ControlPlane": "", "Join": ""}
```

Comment:
I don't think a map is the best way for this. Feels a little fragile. Can we maybe have custom type that encapsulates all this? A simple struct with methods to either populate itself using generated data, or data read from files on disk would be nice.